source/_integrations/logger.markdown

```diff
@@ -146,3 +150,6 @@ the [SSH add-on](/addons/ssh/):
 ```bash
 $ tail -f /config/home-assistant.log
 ```
+
+Docker users can see the logs directly from the host command line with `docker logs my_container_id` - 
```

Comment:
Let's add `--follow` to match the tail command (please note, the last sentence needs to be adjusted for this layout.

````suggestion
Docker users can see the logs directly from the host command line with:
```bash
docker logs --follow  MY_CONTAINER_ID
```
````

Reply:
thanks for suggestion, @frenck - hope you like the rewrite